docs/DeveloperGuide.md

```diff
@@ -459,6 +448,19 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * **Private contact detail**: A contact detail that is not meant to be shared with others
 * **Versioned AddressBook**: An extension to the AddressBook class that allows undoing/redoing actions.
 
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Planned Enhancements**
+**Note:** Our team size is 5
+
+1. **Make the GUI wrap large addresses to the next line:** With the current implementation, if you type in an address which is longer than the width of the screen, it stretches the person card (for all contacts), forcing the user to scroll to the right to see the rest of the address, the project status and deadline. In future updates, we plan to enable `textWrap` and set a `maxWidth` for the address label to enhance the UI and improve readability.
+2. **Add support for patronymics, like 's/o' or 'd/o', in the name field:** The current implementation only allows alphanumeric characters, spaces, and dashes, and so users have to work around this restriction by using "s o" or "son of". In the future, we will adjust the regex to something like this `[\\p{Alnum}]+(([ -][\\p{Alnum}]+)*|(([ -](s|d)/o\\s)[\\p{Alnum}]+))\\s*` to allow special strings like "s/o" and "d/o".
+3. **Make prefix for deadlines `dl/`:** This is in conjunction with point 2. Our current implementation of deadline is implemented with d/date. If we implement a regex to accept d/o, this would trigger an error since it would match d/. In the future, we will change the prefix to something like `PREFIX_DEADLINE = new Prefix("dl/")` 
+4. **Allow users to clear current sorting scheme:** Currently, users cannot 'undo' a sort command. In the future, we plan to add an undo and redo feature, which will complete this enhancement as well. Do refer to [the proposed undo and redo feature](#proposed-undoredo-feature).
+5. **Adjust sorting function to be stable:** With the current implementation of sort, a rather rare bug occurs. When you first sort the list, clients will be sorted correctly based on name/deadline depending on the parameter you chose. Following this, the bug arises when multiple clients have the same name (if you sorted based on name) or same deadline (if you sorted based on deadline). If you change one of the fields of the people with the same name (if you sorted based on name) or same deadline (if you sorted based on deadline), sometimes, the list will reorder among people of the same name/deadline. For future implementations, we plan to modify this such that the list doesn't reorder those with the same name/deadline.
```

Comment:
Can also mention that this is an issue with the `ObservableList` interface (or whatever its called, I forget) we are using, and add a hyperlink this post for the issue: [https://bugs.openjdk.org/browse/JDK-8301761](https://bugs.openjdk.org/browse/JDK-8301761)

Reply:
Yep this is under rahul's PR

Reply:
I think my PR addresses this